frontend/packages/dev-console/integration-tests/features/getting-started/sample-card_Add_page.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Create Sample Application
+    As a user I want to create the Sample Application from Add options
+
+Background:
+    And user is in dev perspecitve
+    And open project namespace 
+
+
+@regression
+Scenario: Sample Card in Add flow
+   Given user is in Add flow of dev perspective
+   When user will see Samples card
```

Comment:
We can remove this statement as it shouldn't be a When statement.

Reply:
It can be modified because it is the first time samples card gets the reference in +Add page